tavern/raws/dungeon.py

```diff
@@ -15,23 +15,23 @@ def __init__(self, name, adjectives=set(), nouns=set(), epithets=set(), monster_
 
 types = {
     DungeonType('cave',
-                adjectives={'dank', 'deep', 'subterranean', 'sunken'},
-                nouns={'cave', 'mines'},
-                epithets={'the underworld', 'the earthmother', 'the mole mother', 'rocks and dirt'},
+                adjectives={'dank', 'deep', 'dripping', 'hollow', 'murky', 'subterranean', 'sunken', 'vast', 'vaulted', 'yawning'},
+                nouns={'burrow', 'cave', 'cavity', 'den', 'hollow', 'karst', 'maw', 'mines', 'sinkhole', 'tube'},
+                epithets={'endless echoes', 'gaping proportions', 'rocks and dirt', 'the bioluminescent', 'the collapsed god', 'the earthmother', 'the mole mother', 'the slimy walls', 'the underworld', 'unknown depth'},
                 monster_reqs={'rat', 'troll'},
                 monster_opts={'ancient', 'giant'},
                 ),
     DungeonType('clockwork',
-                adjectives={'clockwork', 'engineered', 'mechanized'},
-                nouns={'contraption', 'engine', 'machine'},
-                epithets={'endless gears', 'perpetual motion'},
+                adjectives={'clockwork', 'engineered', 'grandfatherly', 'mechanized', 'meticulous', 'precise', 'repetitive', 'soporific', 'swinging', 'ticktock'},
+                nouns={'apparatus', 'clocktower', 'contraption', 'engine', 'factory', 'gizmo', 'gnomon', 'machine', 'plant', 'turbine'},
+                epithets={'endless gears', 'ethereal futures', 'hypnotic rhythm', 'incessant progress', 'perpetual motion', 'ringing bells', 'the conscious cogs', 'the grandfather', 'the grinding gears', 'the twelve hands'},
                 monster_reqs={'automaton', 'golem', 'wind-up'},
                 monster_opts={'quick', 'reconstructed'},
                 ),
     DungeonType('feyland',
-                adjectives={'half-forgotten', 'ever-fading', 'mythic'},
-                nouns={'dream', 'fable'},
-                epithets={'wonders', 'hidden desires', 'the fey', 'wonders'},
+                adjectives={'auroral', 'deceptive', 'ever-fading', 'half-forgotten', 'impish', 'mythic', 'otherworldly', 'shrouded', 'twisted', 'whimsical'},
+                nouns={'court', 'dimension', 'dominion', 'dream', 'fable', 'haven', 'holding', 'kingdom', 'realm', 'vision'},
```

Comment:
All of these feel like they belong better in the "manor" section: court, dominion, holding, kingdom, realm